src/cmd_line/subparser.ts

```diff
@@ -202,6 +203,10 @@ export const commandParsers = {
     parser: parseOptionsCommandArgs,
   },
 
+  sh: {
```

Comment:
This should be `shell` with `abbrev: 'sh'`

Reply:
Yes, this is true. I have changed it now.